### ISSUE TITLE
Cannot preview or view HEIF/HEIC picture format in the main or gallery view

### ISSUE THREAD

**Issue by sebma** [thunderbird-conversations/thunderbird-conversations#2005]
* **Thunderbird version: 102.13.0**
* **Thunderbird Conversation version: 4.0.26**
* **Operating System: Ubuntu 20.04 LTS**

**What did you do?**
Received an email with .HEIC pictures files.

**What did you expect to happen?**
The thumbnails and the picture are viewable in the "Gallery view"

**What actually happened?**
Nothings shows up in the picture frames.

**Comment by Standard8** [comment]
Conversations supports the image formats that Thunderbird supports. Thunderbird is based on Firefox's (the web browser) back end.

Hence it would need to be supported in Firefox/Thunderbird for it to be supported in Conversations.

However from a quick look, there are patenting and licensing concerns about this being implemented for the web, and indeed only Safari supports it - which is not surprising since it is their own custom format. A couple of links below:

https://caniuse.com/?search=heif
https://bugzilla.mozilla.org/show_bug.cgi?id=1402293

In short, I don't see this happening as this is a propriety format.
